uobtheatre/payments/transferables.py

```diff
@@ -0,0 +1,78 @@
+from typing import Optional
+
+from django.db import models
+
+from uobtheatre.payments.payables import Payable, PayableQuerySet
+
+
+class TransferableQuerySet(PayableQuerySet):
+    pass
+
+
+TransferableManager = models.Manager.from_queryset(TransferableQuerySet)
+
+# pylint: disable=abstract-method
+class Transferable(Payable):
+    """
+    A transferable is a payable which can be transfered. A transfer is a
+    to/from another transferable of the same type. For bookings this enables bookings to be updated (chnaged perfomrance)
+    """
+
+    objects = TransferableManager()  # type: ignore
+
+    # Whether the booking has be transferred to
+    transfered_from = models.OneToOneField(
+        "self", on_delete=models.RESTRICT, null=True, related_name="_transferred_to"
+    )
+
+    @property
+    def transferred_to(self) -> Optional["Transferable"]:
+        """The transfered which this has been transfred to.
+
+        If this has not be transfered None will be returned.
+
+        Note: _transferred_to cannot be used directly as if this booking has not
+        been transfered an error is raised.
+        """
+        if not hasattr(self, "_transferred_to"):
+            return None
+        return self._transferred_to  # type: ignore
+
+    @property
+    def transfer_fee(self) -> int:
+        return 200
```

Comment:
We should probably make a constants file for this stuff to live in

Reply:
Are we really making this £2? Do we need to think some more about this...

Reply:
let x = booking price

transfer fee = min( max(100, 0.15*x), 500)    or something